Cargo.toml

```diff
@@ -23,6 +23,7 @@ procfs = "0.15.1"
 serde = { version = "1.0.203", features = ["derive"] }
 serde_json = "1.0"
 serde_repr = "0.1"
+serde_tuple = "1.0.0"
```

Comment:
This isn't in fbsource, but I hope that doesn't cause to much pain ...

I generally check fbsource before adding deps ... just cause of the fun that can bring ...

Reply:
Good point. I'll look into importing it.

Reply:
It should be fine. Third party for rust will actually bring it in for us I think here ... leave it to me.